lib/strategies/webapp-strategy.js

```diff
@@ -344,12 +345,20 @@ function getTokens(req, options, strategy) {
 		code: req.query.code
 	};
 
+	function cb(err, user, info) {
+		if (err) {
+			return strategy.fail;
+		} else {
+			return strategy.success(user || null, info);
+		}
+	}
 	retrieveTokens(formData, strategy).then(function (appIdAuthContext) {
 		// Save authorization context to HTTP session
 		req.session[WebAppStrategy.AUTH_CONTEXT] = appIdAuthContext;
+		let user = appIdAuthContext.identityTokenPayload.sub;
 		delete req.session[WebAppStrategy.STATE_PARAMETER];
 		logger.debug("completeAuthorizationFlow :: success", options);
-		strategy.success(appIdAuthContext.identityTokenPayload || null);
+		strategy._verify(user, cb);
```

Comment:
Tal suggests sending `accessToken`, `refreshToken`, `IDToken`, and `cb` to verify